projects/storefrontlib/shared/components/form/file-upload/file-upload.component.html

```diff
@@ -16,9 +16,9 @@
   <ng-content></ng-content>
 </button>
 <!-- TODO(Brian-Parth): CXEC-17196 -->
-<div *ngIf="customButton" (click)="fileInput.click()">
+<button class="link" *ngIf="customButton" (click)="fileInput.click()">
```

Comment:
action links should also have this class `class= "link cx-action-link"`

Reply:
Got it